src/views/DatasetManager/components/DatasetList/DatasetList.js

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React, { useCallback, useState } from 'react';
+import { Box, Divider, IconButton, List, ListItem, ListItemButton, ListItemText, Typography } from '@mui/material';
+import { DeleteForever as DeleteForeverIcon, Refresh as RefreshIcon } from '@mui/icons-material';
+import { DontAskAgainDialog } from '@cosmotech/ui';
+import { CreateDatasetButton } from '../../../../components';
```

Comment:
Maybe `CreateDatasetButton` should be a sub-component of `DatasetList`, if it is only used here?

Reply:
Yes, you are right, I put it in _/components_ with `CreateScenarioButton` but it is used in only place :slightly_smiling_face: